src/node/rpc/forwarder.h

```diff
@@ -80,24 +146,47 @@ namespace ccf
       }
       serialized::write(data_, size_, raw_request.data(), raw_request.size());
 
-      ForwardedHeader msg = {
-        ForwardedMsg::forwarded_cmd, rpc_ctx->frame_format()};
+#if false
```

Comment:
`?>_>`

Reply:
Minimising the diff between this PR, and the one that actually enables the behaviour

Reply:
Seems to be a small diff anyway? Either way, we probably shouldn't resort to `if false` :)

Reply:
Sure, I've removed this block.